dev/devicelab/manifest.yaml

```diff
@@ -124,6 +124,13 @@ tasks:
     stage: devicelab
     required_agent_capabilities: ["mac/android"]
 
+  run_release_test:
+    description: >
+      Checks that `flutter run --release` does not crash.
+    stage: devicelab
+    required_agent_capabilities: ["mac/android"]
+    flaky: true
```

Comment:
Why flaky?

Reply:
file says to always add tests with flaky:true initially, to test it on devicelab as a dry run before exposing everyone else to the test.